index.js

```diff
@@ -51,21 +50,22 @@ exports.decorateConfig = config => {
         border-color: transparent !important;
       }
       .tab_tab {
+        border: transparent !important;
         color: ${foregroundColor} !important;
-        background-color: ${backgroundColor};
+        background-color: #e6dfcb;
+        border-bottom: solid 3px ${activeTabBorderColor} !important;
       }
       .tabs_title {
         color: ${foregroundColor} !important;
       }
-      .active_1gcgehd:before {
+      .tab_tab.tab_active:before {
```

Comment:
@rauchg told me that `active_1gcgehd` was not a dependable selector, as it is likely to change in between HyperTerm releases.